src/js/page.js

```diff
@@ -145,9 +147,9 @@
       let nodeSelector = "#newtab-grid, #newtab-search-container";
       for (let node of document.querySelectorAll(nodeSelector)) {
```

Comment:
@marcoscaceres  can't use forEach on querySelectorAll...not an array! :(
could do something like 
````
var nodeList = document.querySelectorAll("#newtab-grid, #newtab-search-container"); 
var nodeArray = Array.prototype.slice.call(nodeList);
nodeArray.forEach(node => node.dataset.pageDisabled = !aValue);
````
thoughts?